src/components/PlayButton.js

```diff
@@ -20,12 +20,7 @@ export default function PlayButton({ isPlaying, playOrPause }) {
       </Button> : <Button variant="link" onClick={() => playOrPause()}>
       <FaPlayCircle size={30}/>
       </Button>}
-      
-      
-
-      <audio className="audio-element">
-        <source src="http://boombox.middlebury.edu:8000/" type="audio/mpeg" />
-      </audio>
```

Comment:
This part of the code does not work. We are planning to speak to the professor about that and switch it to iframe instead of audio tags. it is really inconsequential now. I can keep it in there, but it is going to be changed anyway

Reply:
Ok that makes sense, I don't think there's any need for you to change it. Thanks for the explanation and LGTM.